### ISSUE TITLE
Incorrect error count in Status Bar

### ISSUE THREAD

**Issue by rahulthewall** [steelbrain/linter-ui-default#338]
See the screenshot attached below, I see a large number of linter errors, but these errors do not exist. So where is this count coming from?

<img width="1680" alt="screen shot 2017-05-29 at 11 38 31" src="https://cloud.githubusercontent.com/assets/2026429/26544783/6b0d08d0-4463-11e7-92aa-73b9237a4d92.png">

Atom Version: 1.17.2
OS: OS X 10.12.5

Lint related packages
```
├── linter@2.1.4
├── linter-eslint@8.2.1
├── linter-flow@5.6.0
├── linter-ui-default@1.6.0
```

Could someone tell me what is happening? Dev tools reports no issue.

**Comment by steelbrain** [comment]
Hi @rahulthewall 

If you look at your configuration of `StatusBar Represents`, you'll see that it's representing all of the project. You can change it to represent a single file if you want. Alternatively, you could make your panel represent the Entire Project as well

**Comment by rahulthewall** [comment]
Hi @steelbrain, thanks a lot. Should have looked into the settings properly. :)
